### ISSUE TITLE
ADD tar does not work.

### ISSUE THREAD

**Issue by xingao267** [GoogleContainerTools/kaniko#700]
**Actual behavior and how to reproduce**
ADD tar files in Dockerfile does not work properly with Kaniko.

Dockerfile content
```
FROM alpine:3.9
ENTRYPOINT ["sh"]
ADD tarfile.tar /
```
tarfile.tar is simple tarball that contains a test.txt file inside.

Command I run:
```
docker run -v <path_to_my_local_context_dir>:/workspace gcr.io/kaniko-project/executor:latest --dockerfile Dockerfile --context /workspace --reproducible --no-push --tarPath=/workspace/image.tar -d "someimage"

docker load -i image.tar

docker run -it someimage
```
The content of tarfile.tar is not unpacked and made available at / in the container.

**Expected behavior**
There should be a file named test.txt exist in the / in the container.

**Additional Information**
 - Dockerfile
See above

 - Build Context
   Please provide or clearly describe any files needed to build the Dockerfile (ADD/COPY commands) 
Create a simple tarball that contains a txt file, and use that in build context.

 - Kaniko Image (fully qualified with digest)
gcr.io/kaniko-project/executor@sha256:d9fe474f80b73808dc12b54f45f5fc90f7856d9fc699d4a5e79d968a1aef1a72

**Comment by priyawadhwa** [comment]
Hey @xingao267 thanks for submitting this issue. I just merged #717, could you see if this solves this problem? You can run `gcr.io/kaniko-project/executor:0694be3e85e127ed597fdb068b1f91c4478806f8` to try the latest version of the image.

**Comment by tejal29** [comment]
This is fixed now.
